server/app.js

```diff
@@ -1,21 +1,29 @@
-const express = require("express");
-const morgan = require("morgan");
-const cors = require("cors");
-const bodyParser = require("body-parser");
-const mongoose = require("mongoose");
+const express = require("express")
+const morgan = require("morgan")
+const cors = require("cors")
+const bodyParser = require("body-parser")
+const app = express()
+ 
+//parsers
+app.use(cors())
+app.use(morgan("dev"))
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
 
-const app = express();
+// server client files
+app.use(express.static(__dirname + "/../client/build"))
+app.get('/', (req, res)=>{
+   res.sendFile('/../client/build/index.html')
+})
 
-const routes = require("./api/routes/");
+// redirector
+app.get('*', (req, res) => {
+  res.redirect('/')
+})
```

Comment:
Instead of redirecting to root, let's give 404.

Reply:
ok

Reply:
done